webrtc.html

```diff
@@ -1516,77 +1529,101 @@
+                    when <code>createAnswer</code> was invoked, wait for
+                    <a href="#sec.identity-proxy-assertion-request">the
+                    identity assertion request process</a> to complete.</p>
+                  </li>
+                  <li>
+                    <p>If the identity provider was unable to produce an
+                    identity assertion, reject <var>p</var> with a
+                    <code>DOMException</code> object whose <code>name</code>
+                    attribute has the value <code>NotReadableError</code>, and
+                    abort these steps.</p>
+                  </li>
+                  <li>
+                    <p>If <var>connection</var> was not constructed with a set
+                    of certificates, and one has not yet been generated, wait
+                    for it to be generated.</p>
+                  </li>
+                  <li>
+                    <p>Inspect the system state to determine the currently
+                    available resources as necessary for generating the answer,
+                    as described in <span data-jsep=
+                    "createanswer">[[!JSEP]]</span>.</p>
+                  </li>
+                  <li>
+                    <p>If this inspection failed for any reason, reject
+                    <var>p</var> with a <code>DOMException</code> object whose
+                    <code>name</code> attribute has the value
+                    <code>OperationError</code>, and abort these steps.</p>
+                  </li>
+                  <li>
+                    <p>Queue a task that runs the <a>final steps to create an
+                    answer</a>, given <var>p</var>.</p>
+                  </li>
+                </ol>
+                <p>The <dfn>final steps to create an answer</dfn> given a
+                promise <var>p</var> are as follows:</p>
+                <ol>
+                  <li>
+                    <p>If <var>connection</var>'s [[<a>isClosed</a>]] slot is
+                    <code>true</code>, then abort these steps.</p>
+                  </li>
+                  <li>
+                    <p>If <var>connection</var> was modified in such a way that
+                    additional inspection of the system state is necessary,
+                    then in parallel begin the <a>steps to create an answer</a>
+                    again, given <var>p</var>, and abort these steps.</p>
+                    <div class="note">
+                      This may be necessary if, for example,
+                      <code>createAnswer</code> was called when an
+                      <code><a>RTCRtpTransceiver</a></code>'s direction was
+                      "recvonly", but while performing the <a>steps to create
+                      an answer</a> in parallel, the direction was changed to
+                      "sendrecv", requiring additional inspection of video
+                      encoding resources.
```

Comment:
This is a much more niche situation than `createOffer`'s. But I figured it was worth including for symmetry, if nothing else.